df.history.xml

```diff
@@ -2279,29 +2279,30 @@
       <int32_t name='entity' ref-target='historical_entity' comment='the guild to which the figure belongs?'/>
       <int32_t name='source_site' ref-target='world_site'/>
       <int32_t name='dest_site' ref-target='world_site'/>
-      <int32_t/>
-      <int32_t/>
-      <int32_t/>
-      <int32_t comment='value of the trade?'/>
+      <int32_t name='production_zone'/>
+      <int32_t name='allotment'/>
+      <int32_t name='allotment_index'/>
+      <int32_t name='account_shift'/>
     </class-type>
 
     <class-type type-name='history_event_add_entity_site_profile_flagst' inherits-from='history_event' since='v0.47.01'>
       <int32_t name='entity' ref-target='historical_entity'/>
       <int32_t name='site' ref-target='world_site'/>
-    </class-type>
-
-    <class-type type-name='history_event_add_hf_entity_honorst' inherits-from='history_event' since='v0.47.01'>
-      <int32_t name='entity' ref-target='historical_entity'/>
-      <int32_t name='hf' ref-target='historical_figure'/>
-      <int32_t comment='index or identifier for the honor?'/>
+      <int32_t comment='always 65536, possibly should be two 16-bit ints?'/>
```

Comment:
Even if we haven't figured out what this represents yet, I think it's fairly safe to say it should be split into two `<int16_t>`s at this point

Reply:
Will do.